docs/runbooks/CDIDataImportCronOutdated.md

```diff
@@ -69,6 +71,15 @@ $ kubectl get dataimportcron cron-test -o yaml | grep -B 5 storageClassName
         storageClassName: rook-ceph-block
 ```
 
+When no storage class is defined, DataVolume controller will not create PVCs, and we'll get the event:
+`DataVolume.storage spec is missing accessMode and no storageClass to choose profile`  
+
+Once a default storage class has been defined, updated versions of CDI should resolve this issue automatically. If it does not resolve within a few seconds, manually delete the affected DataVolumes. CDI will then recreate them with the proper reference to the default storage class.
+
+The affected DataVolumes can be found by:  
+```bash
+$ kubectl get dv -A | awk '{if ($4 == "" || NR == 1) {print}}'`
```

Comment:
Should this be all namespaces or just openshift-virtualization-os-images?  I'd hate for someone to mistakenly delete another DV.